app/src/main/java/com/google/droidjump/LevelsAdapter.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.droidjump;
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+import android.graphics.Color;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.TextView;
+
+public class LevelsAdapter extends BaseAdapter {
+
+    private Context context;
+    private LayoutInflater inflater;
+    private int[] levels;
+    private int currentLevel;
+
+    public LevelsAdapter(Context context, int[] levels, int currentLevel) {
+        this.context = context;
+        this.levels = levels;
+        this.currentLevel = currentLevel;
+    }
+
+    @Override
+    public int getCount() {
+        return levels.length;
+    }
+
+    @Override
+    public Object getItem(int i) {
+        return levels[i];
+    }
+
+    @Override
+    public long getItemId(int i) {
+        return i-1;
```

Comment:
why i-1? 
I am not sure what this is supposed to be.

Reply:
Should be i+1 or levels[i]. We have array of levels. i - index. We start counting indexes from 0 and levels - from 1. So levels[0] = 1, levels[1] = 2.

Reply:
Item - level. id - index. For example, levels[0] = 1, levels[3] = 4. If we want to find id of level 4, it will be 3. So id = level - 1. Will write a small comment for this method.